src/util/tubemap.js

```diff
@@ -682,10 +684,32 @@ function placeReads() {
           previousValidY = reads[idx].path[lastIndex].y;
           lastIndex = lastIndex - 1;
         }
+
+        // sometimes, elements without nodes are between 2 segments going to the same node, from the same direction
+        // this means we're looping back to the same node, which constitutes a different sorting priority
```

Comment:
I'm not sure "constitutes" is the right word here. Do you mean we want to use a different sorting in this case?